sdk/communication/Azure.Communication.Chat/README.md

```diff
@@ -272,43 +272,43 @@ Use `DeleteMessage` to delete a message.
-AsyncPageable<ChatThreadMember> allMembers = chatThreadClient.GetMembersAsync();
-await foreach (ChatThreadMember member in allMembers)
+```C# Snippet:Azure_Communication_Chat_Tests_Samples_GetParticipants
+AsyncPageable<ChatParticipant> allParticipants = chatThreadClient.GetParticipantsAsync();
+await foreach (ChatParticipant participant in allParticipants)
 {
-    Console.WriteLine($"{member.User.Id}:{member.DisplayName}:{member.ShareHistoryTime}");
+    Console.WriteLine($"{participant.User.Id}:{participant.DisplayName}:{participant.ShareHistoryTime}");
 }
 ```
-### Add thread members
+### Add thread participants
 
-Use `AddMembers` to add members to the chat thread. The following are the supported attributes for each thread member:
-- `communicationUser`, required, it is the identification for the thread member.
-- `displayName`, optional, is the display name for the thread member.
-- `shareHistoryTime`, optional, time from which the chat history is shared with the member.
+Use `AddParticipants` to add participants to the chat thread. The following are the supported attributes for each thread participant:
+- `communicationUser`, required, it is the identification for the thread participant.
+- `displayName`, optional, is the display name for the thread participant.
+- `shareHistoryTime`, optional, time from which the chat history is shared with the participant.
 
-```C# Snippet:Azure_Communication_Chat_Tests_Samples_AddMembers
-var members = new[]
+```C# Snippet:Azure_Communication_Chat_Tests_Samples_AddParticipants
+var participants = new[]
 {
-    new ChatThreadMember(new CommunicationUser(memberId1)) { DisplayName ="display name member 1"},
-    new ChatThreadMember(new CommunicationUser(memberId2)) { DisplayName ="display name member 2"},
-    new ChatThreadMember(new CommunicationUser(memberId3)) { DisplayName ="display name member 3"}
+    new ChatParticipant(new CommunicationUser(participantId1)) { DisplayName ="display name participant 1"},
+    new ChatParticipant(new CommunicationUser(participantId2)) { DisplayName ="display name participant 2"},
+    new ChatParticipant(new CommunicationUser(participantId3)) { DisplayName ="display name participant 3"}
 };
-await chatThreadClient.AddMembersAsync(members);
+await chatThreadClient.AddParticipantsAsync(participants);
 ```
 
-### Remove thread member
+### Remove thread participant
 
-Use `RemoveMember` to remove a thread member from the thread.
-`communicationUser` is the identification of the chat member.
+Use `RemoveParticipant` to remove a thread participant from the thread.
+`communicationUser` is the identification of the chat participant.
 
-```C# Snippet:Azure_Communication_Chat_Tests_Samples_RemoveMember
-await chatThreadClient.RemoveMemberAsync(new CommunicationUser(memberId));
+```C# Snippet:Azure_Communication_Chat_Tests_Samples_RemoveParticipant
+await chatThreadClient.RemoveParticipantAsync(new CommunicationUser(participantId));
```

Comment:
you mean something like this instead ?
var participant = new CommunicationUser(participantId); //This won't show in the readme 

await chatThreadClient.RemoveParticipantAsync(participant);

Reply:
I do not think we even need to do `new CommunicationUser(participantId)`. When we use the `communicationIdentityClient.CreateUserAsync` method, it will give us the user. Right now we create a user, get the id out of it and then create a user object again from the that id. Looking at the participant sample code. It almost does:

```C#
Response<CommunicationUser> user1 = await communicationIdentityClient.CreateUserAsync();

CommunicationUserToken communicationUserToken1 = await communicationIdentityClient.IssueTokenAsync(user1.Value, new[] { CommunicationTokenScope.Chat });

string theadCreatorUserId = communicationUserToken1.User.Id;

var participantId1 = theadCreatorUserId;

var participants = new[]
{
  new ChatParticipant(new CommunicationUser(participantId1)) { DisplayName ="display name participant 1"},
  new ChatParticipant(new CommunicationUser(participantId2)) { DisplayName ="display name participant 2"},
  new ChatParticipant(new CommunicationUser(participantId3)) { DisplayName ="display name participant 3"}
};
```

Instead we can just always just use the CommunicationUser that comes out of the previous API calls. Eg for creating a thread, we could define the scenario as "Dalia wants to start a chat conversation with Robin and Lyla about their launch meeting":

```C#
Response<CommunicationUser> createDaliaUserResponse = await communicationIdentityClient.CreateUserAsync();
CommunicationUser dalia = createDaliaUserResponse.Value;

// Or we can even switch to sync APIs and say:
// CommunicationUser dalia = communicationIdentityClient.CreateUser().Value;

Response<CommunicationUser> createRobinUserResponse = await communicationIdentityClient.CreateUserAsync();
CommunicationUser robin = createRobinUserResponse.Value;

Response<CommunicationUser> createLylaUserResponse = await communicationIdentityClient.CreateUserAsync();
CommunicationUser lyla= createLylaUserResponse.Value;

var participants = new[]
{
  new ChatParticipant(dalia) { DisplayName ="Dalia"},
  new ChatParticipant(robin) { DisplayName ="Robin"},
  new ChatParticipant(lyla) { DisplayName ="Lyla"}
};
```

And when we want to remove we will just use the same user objects:
```C#
chatThreadClient.RemoveParticipant(robin);
```